src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapper.java

```diff
@@ -143,6 +143,11 @@
         return errorDetailsResponseEntity(ex, FORBIDDEN, ACCESS_EXCEPTION.getErrorMessage());
     }
 
+    @ExceptionHandler(RequestRejectedException.class)
+    public ResponseEntity<Object> handleRequestRejectedException(RequestRejectedException ex) {
```

Comment:
probably we dont need this exceptionhandler anymore

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/1ca60a5369b53bae237ab6c14a9b9df1c4e83cf7